project/spotify/static/spotify/neighbors.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	
+	$.getJSON("/neighborsinfo",function(data) {
+		var users = data['user_list']
+		var neighbors = data['neighbors_list']
+		for (user_index in users) {
+			$("#neighbors").append(users[user_index] +"<ol>")
```

Comment:
Identifier 'user_index' is not in camel case.<br>Missing semicolon.<br>'user_index' is not defined.